src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPLIT_CHARACTER = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        File toFile = new File(toFileName);
+        StringBuilder builder = new StringBuilder();
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFile))) {
+            String value = reader.readLine();
+
+            while (value != null) {
+                builder.append(value).append(SPLIT_CHARACTER);
+                value = reader.readLine();
```

Comment:
better append line separator as there is a case when there will be commas in line already, and your code will be confused and produce unexpected result
```suggestion
                builder.append(value).append(System.lineSeparator());
```